### PR TITLE
Fix DynamicView fabric link

### PR DESCRIPTION
**Pull request by M4rtinOF**
Fixes the curseforge link of DynamicView in the fabric section, wich previosly redirected to the forge version of the mod

### CODE REVIEW

File: Performance/Performance119.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Don't want to mess around with mods? try out Fabulously Optimized [![Modrinth Lo
...
-| [Dynamic View](https://www.curseforge.com/minecraft/mc-mods/dynamic-view) [![CurseForge Logo](/images/platform_icons/CurseForge.png)](https://www.curseforge.com/minecraft/mc-mods/dynamic-view)[![Github Logo](/images/platform_icons/Github.png)](https://github.com/ldtteam/DynView) | APTweaks | This is a small/light serverside utility mod to help balancing lag (TPS) and chunk view/load distance. | someaddon | Server | none | [LGPL-3.0](/licenses/Licenses.md#lgpl-30)
+| [Dynamic View](https://beta.curseforge.com/minecraft/mc-mods/dynamic-view-fabric/files) [![CurseForge Logo](/images/platform_icons/CurseForge.png)](https://beta.curseforge.com/minecraft/mc-mods/dynamic-view-fabric/files)[![Github Logo](/images/platform_icons/Github.png)](https://github.com/ldtteam/DynView) | APTweaks | This is a small/light serverside utility mod to help balancing lag (TPS) and chunk view/load distance. | someaddon | Server | none | [LGPL-3.0](/licenses/Licenses.md#lgpl-30)
```
Comment (lrmtheboss) [inline Performance/Performance119.md:35]: Why was this changed from www to beta?

**Review by lrmtheboss — CHANGES_REQUESTED** [review]
The link does not need /files on the end and should be www. not beta.

File: Performance/Performance119.md (hunk 1 above)
Comment (M4rtinOF) [reply Performance/Performance119.md]: Oh ok, sorry for that, for some reason the curseforge links we're redirecting me to the beta ones

**Comment by M4rtinOF** [comment]
Changed the links from beta to normal

**Comment by M4rtinOF** [comment]
Changed the links from beta to www

**Review by lrmtheboss — CHANGES_REQUESTED** [review]
Still need to remove /files from the end of the url

**Comment by lrmtheboss** [comment]
closed as a duplicate of newer #209
